cmd/param.go

```diff
@@ -41,22 +41,43 @@ func init() {
 	paramDiffCmd.PersistentFlags().String(flagParamComponent, "", "Specify the component to diff against")
 }
 
+var paramShortDesc = map[string]string{
+	"set":  `Change component or environment parameters (e.g. replica count, name)`,
+	"list": `List known component parameters`,
+	"diff": `Display differences between the component parameters of two environments`,
+}
+
 var paramCmd = &cobra.Command{
 	Use:   "param",
-	Short: `Manage ksonnet component parameters`,
-	Long: `Parameters are the customizable fields defining ksonnet components. For
-example, replica count, component name, or deployment image.
-
-Parameters are also able to be defined separately across environments. Meaning,
-this supports features to allow a "development" environment to only run a
-single replication instance for it's components, whereas allowing a "production"
-environment to run more replication instances to meet heavier production load
-demands.
-
-Environments are ksonnet "named clusters". For more information on environments,
-run:
-
-    ks env --help
+	Short: `Manage ksonnet parameters for components and environments`,
+	Long: `
+Parameters are customizable fields that are used to expand and define ksonnet
+*components*. Examples might include a deployment's 'name' or 'image'. Parameters
+can also be defined on a *per-environment* basis. (Environments are ksonnet
+deployment targets, e.g. specific clusters. For more info, run ` + "`ks env --help`" + `.)
+
+For example, this allows a ` + "`dev`" + ` and ` + "`prod`" + ` environment to use the same component
+manifest for an nginx deployment, but customize ` + "`prod`" + ` to use more replicas to meet
+heavier load demands.
+
+Params are structured as follows:
+
+* App params (stored in ` + "`components/params.libsonnet`" + `)
+    * Component-specific params
+        * Originally populated from ` + "`ks generate`" + `
+        * e.g. 80 for ` + "`deployment-example.port`" + `
+    * Global params
+        * Out of scope for CLI (requires Jsonnet editing)
+        * Use to make a variable accessible to multiple components (e.g. service name)
+
+* Per-environment params (stored in + ` + "`environments/<env-name>/params.libsonnet`" + `)
+    * Component-specific params ONLY
+    * Override app params (~inheritance)
+
+Note that all of these params are tracked **locally** in version-controllable
+Jsonnet files.
+
+----
```

Comment:
Do you want to keep this consistent with the other commands and use `### Syntax`?

Reply:
discussed offline: so I'm using `---` instead of `### Syntax`  for commands that don’t actually run anything